### ISSUE TITLE
Use colorama for Windows support

### ISSUE THREAD

**Issue by hugovk** [hugovk/norwegianblue#8]
@hugovk small suggestion :
add `colorize` into your install_requires and also make small changes  to your src/norwegianblue/__init__.py  : 
change :
```python
from termcolor import colored
```
to
```python
from colorama import init
from termcolor import colored
init()
```
this will let color also works on windows systems
termcolor is really old module and doesnt support some systems so using colorama and termcolor together
will fix all problems.

_Originally posted by @usta in https://github.com/endoflife-date/endoflife.date/issues/273#issuecomment-882052650_

---

Will make it a conditional install for Windows only.

**Comment by hugovk** [comment]
I tested the currently released version with Python 3.9.7 on Windows 10 and colours are showing in the default Command Prompt:

![image](https://user-images.githubusercontent.com/1324225/133047717-b649ac48-1924-4235-ba8e-0feaf8eb4495.png)

**Comment by hugovk** [comment]
Doesn't seem necessary, can re-open if needed in the future.
